### DISCUSSION TITLE
Intro video is private can i get a public version

### DISCUSSION THREAD

**Discussion by buddikag** [eclipse-birt/birt#1415] [Q&A]
https://www.youtube.com/watch?v=6TRndgUMUmU this video tutorial in not working. Can I get public version of this video, Watch [tutorial 1] and  2

**Comment by speckyspooky** [comment]
This is not a tecnical issue of birt or the birt web page. There are nor further information what the conntent of the video should be.
Therefore I switch it to discussion.
